ci-operator/config/red-hat-storage/odf-console/red-hat-storage-odf-console-master.yaml

```diff
@@ -1,6 +1,20 @@
+base_images:
+  base:
+    name: release
+    namespace: openshift
+    tag: tectonic-console-builder-v23
 build_root:
-  project_image:
-    dockerfile_path: Dockerfile.ci
+  image_stream_tag:
+    name: release
+    namespace: openshift
+    tag: tectonic-console-builder-v23
```

Comment:
Why is the base image and the build_root declared as same?

Reply:
build_root here is initialised with an empty console image, It does not do anything here in our case but it is a mandatory property for the config.
Base is the image that we are using to build the odf-console image which will have our PR code.